SU-SWS/react_paragraphs
# Fix default values in autocomplete and allow special characters in text fields

# READY FOR REVIEW

# Summary
- Fixed autocomplete widget when edit an existing item and when you remove all selected values
- Fixed all text widgets that use special characters like `+` or `()`

# Need Review By (Date)
- 10/20

# Urgency
- medium

# Steps to Test
1. Checkout this branch
1. clear browser caches
1. create a paragraph type with an entity reference field that allows more than 1 item
1. create a page with that paragraph and populate it
1. go back to edit it and verify if you remove your selections it doesn't blow up.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
